src/token/src/api/inspect.rs

```diff
@@ -36,7 +37,7 @@ static OWNER_METHODS: [&str; 8] = [
     "setOwner",
 ];
 
-static TRANSACTION_METHODS: [&str; 3] = ["approve", "burn", "transfer"];
+static TRANSACTION_METHODS: [&str; 4] = ["approve", "burn", "transfer", "transferAndNotify"];
```

Comment:
Rather than hard coding the size this could be rewritten as 
```rust
static TRANSACTION_METHODS: &[&str] = &["approve", "burn", "transfer", "transferAndNotify"];
```
so adding / removing entries doesn't mean updating the size as well.

Reply:
In addition to this, I think the transfer and notify will be deprecated. 
It would seem we need a guard on transfer_from , but with the balance checking on the "from" field of the argument if we have access to the input arguments.

Reply:
I've applied Jonas's comment, and added check for the `transferFrom` mehtod. @blutooth, shall I remove `transferAndNotify` completely at this time, or will we do it later?